src/commands/help.ts

```diff
@@ -34,6 +40,52 @@ export default {
         const command: Command =
             client.commands.get(name) || client.commands.find((cmd) => cmd.aliases?.includes(name));
 
+        if (name === 'admin') {
+            const embed = new Discord.MessageEmbed()
+                .setColor(0x6293f5)
+                .setAuthor(`${client.user.username} Help`, client.user.displayAvatarURL({ format: 'png' }))
+                .setTitle("Here's a list of all my commands:")
+                .setTimestamp()
+                .setFooter(`You can send "${prefix}help [command name]" to get info on a specific command!`);
+
+            client.commands.forEach((command) => {
+                if (command.level === 'admin') {
```

Comment:
In order to avoid using levels, which I described in my above comments, this suggestion could be implemented:

```suggestion
                if (command.adminOnly) {
```